client/allocrunner/taskrunner/logmon_hook.go

```diff
@@ -94,21 +98,29 @@ func reattachConfigFromHookData(data map[string]string) (*plugin.ReattachConfig,
 
 func (h *logmonHook) Prestart(ctx context.Context,
 	req *interfaces.TaskPrestartRequest, resp *interfaces.TaskPrestartResponse) error {
+	var reattachConfig *plugin.ReattachConfig
+	var err error
 
-	reattachConfig, err := reattachConfigFromHookData(req.PreviousState)
-	if err != nil {
-		h.logger.Error("failed to load reattach config", "error", err)
-		return err
-	}
+	// If the task was restarted then the logmon process is still running and just
```

Comment:
Don't quite follow this comment, Is it meant to be:

```suggestion
	// If the task was restarted but the logmon process is still running, then just
```